lib/ansible/modules/assert.py

```diff
@@ -41,20 +41,29 @@
     default: no
     version_added: "2.8"
 extends_documentation_fragment:
-- action_common_attributes
+  - action_common_attributes
+  - action_common_attributes.conn
+  - action_common_attributes.flow
 attributes:
     action:
         support: full
     async:
         support: none
+    become:
+        support: none
+    bypass_host_loop:
+        support: none
     connection:
         support: none
+    check_mode:
+        support: none
+    diff_mode:
+        support: none
     delegation:
         support: none
-    proprietary:
-        support: full
-    windows:
-        support: full
+        details: Aside from ``register`` and/or in combination with ``delegate_facts``, it has little effect.
```

Comment:
```suggestion
        details: Aside from C(register) and/or in combination with C(delegate_facts), it has little effect.
```
Or is there a reason not to use the usual documentation formatting specifiers?

In any case, this is also something that can easily be changed later.

Reply:
Since `M(...)` is used in other places, I think there is no reason to not use them everywhere.